README.md

```diff
@@ -15,17 +15,35 @@ NetConnect is a desktop app for managing contacts in SMEs, optimized for use via
     * Add a new profile: Easily create a new profile for an employee, client, or supplier. You can choose to tag them with a custom tag.
     * Delete or Edit a profile: Remove a profile from the list or Edit their information via their ID.
 
+
 2. Category Management
     * Roles: Categorize your profiles into *Employees, Clients, or Suppliers*.
+    * Tailored profile information: Each of the roles have their own set of information to be filled in (Employees have the field `Department`, while Supplier have `Product`).
 
 3. Search
+The searches can be stacked and allows you to find a profile based on the following criteria:
```

Comment:
Require an empty line before this for line break.

Reply:
Looks like adding the line break was not a good idea 😆 Indentation gets messed up + I'm missing another line break before the unordered list.
![image](https://github.com/AY2324S2-CS2103T-F12-1/tp/assets/93519596/b9058cc3-a9ba-436b-92de-46fe828d27e8)


Will change to a fullstop instead...